src/Mod/TechDraw/Gui/ViewProviderPage.h

```diff
@@ -45,11 +45,17 @@ class DrawTemplate;
 namespace TechDrawGui
 {
 
+// ?? is changing the base type worthwhile here?  lint thinks so.
+enum PageEditMode : std::int16_t {
+    ShowDrawing = 10,
+    ToggleUpdate = 11 };
+

```

Comment:
I also find the linter a bit agressive. "Used int of 4 bytes instead of size_t of 1 byte", omg 3 bytes!

It's not wrong... but I just tend to ignore it

However, we need to make this `enum class`, else it will convert implicitly to int.

Changing it will therefore also require adding `static_cast<PageEditMode>` everywhere - so depending on how widely it is used, it might be out of scope for this PR.

Reply:
converting this to int isn't a problem as these 2 tokens are only used to compare to ints.

Reply:
> used to compare to ints

Well, that is the problem. Some places an enum class is used, and other times an int is used.

Will be another PR.